scripts/install-previous-hoot-version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+HOOT_RPM_PREFIX="hootenanny-autostart"
+PREVIOUS_VERSION=`sudo yum list --showduplicates hootenanny-autostart | \
```

Comment:
It might be a good idea to check if PREVIOUS_VERSION is empty.  It shouldn't ever be, but it would be an indication that something unexpected happened.  In this case, maybe we error out and post an appropriate error message so it's easy to tell what went wrong.

Reply:
Ok, I can add that